types/use-deep-compare-effect/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for use-deep-compare-effect 1.2
+// Project: https://github.com/kentcdodds/use-deep-compare-effect#readme
+// Definitions by: JPeer264 <https://github.com/JPeer264>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { useEffect } from 'react';
+
+export const useDeepCompareNoEffect: typeof useEffect;
```

Comment:
I don't see this in the [package](https://unpkg.com/use-deep-compare-effect@1.2.0/dist/use-deep-compare-effect.cjs.js).

Reply:
True, I misread the function and updated it to `useDeepCompareEffectNoCheck`